docs/reference/index-modules/indexing-pressure.asciidoc

```diff
@@ -0,0 +1,56 @@
+[[index-modules-indexing-pressure]]
+== Indexing Pressure
+
+Indexing documents into Elasticsearch introduces system load in the form of
+memory and CPU load. Each indexing operation includes coordinating, primary, and
+replica stages. These stages can be performed across multiple nodes in the
+cluster. If too much indexing work is introduced into the system, the cluster
+can become saturated.
```

Comment:
maybe add that it would adversely affect other parts of the system (e.g. searches, cluster coordination, ...)